subsys/bluetooth/host/smp.c

```diff
@@ -284,6 +284,10 @@ static bool sc_oobd_present;
 static bool legacy_oobd_present;
 static bool sc_supported;
 static const uint8_t *sc_public_key;
+static void bt_smp_pkey_ready(const uint8_t *pkey);
+static struct bt_pub_key_cb sc_local_pub_key_cb = {
+	.func           = bt_smp_pkey_ready,
```

Comment:
Probably a consequence of copy-paste from somewhere else, but seems unnecessary to have exactly 11 spaces between `func` and `=`